docs/conf.py

```diff
@@ -53,9 +53,9 @@
 # Taken from qctrl/docs, see head.html in the repo
 html_context = {
     "var_url": "https://docs.q-ctrl.com",
-    "var_title": "Open Controls Python package | Q-CTRL",
+    "var_title": "Open Controls Python",
```

Comment:
Do we still need the Python here?

Reply:
Oops, no we don't, that was my oversight. Fixed now